browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -1,7 +1,7 @@
-/* Copyright 2019 The Brave Authors. All rights reserved.
+/* Copyright 2022 The Brave Authors. All rights reserved.
```

Comment:
pls don't change the year in existing files.

Reply:
`presubmit` complains this when I fix only below `https`.

Reply:
> `presubmit` complains this when I fix only below `https`.

it complains about a missing `(c)` thingy ;)

I'll update the message to not include the current year in "expected message" for existing files.